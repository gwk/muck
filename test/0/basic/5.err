~ muck note: b.txt.py: source mtime changed but contents did not; reverting to: .+
