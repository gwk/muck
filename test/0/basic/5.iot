{
  'desc': 'touch b.txt.py; rebuild reverts mtime.',
  'compile': ['touch b.txt.py'],
  'cmd': '$MUCK -no-times',
  'code': 0,
  'err_mode': 'match',
}
