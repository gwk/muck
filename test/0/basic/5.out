
d.txt[m (â‡  [31mb.txt[m, [31mc.txt[m)
â”—â•¸[32md.txt.py[m
  â•°â•´[34mbasic_lib.py[m

index.html[m
â”—â•¸[32mindex.html.wu[m
  â•°â•´[34ma.txt[m
  â•°â•´[34mb.txt[m
  â”‚ â”¡â•¸[32mb.txt.py[m
  â”‚ â””â•´[31md.txt[m â‡¡
  â•°â•´[34mc.txt[m
  â”‚ â”¡â•¸[32mc.txt.py[m
  â”‚ â”‚ â”—â•¸[32mc.txt.py.py[m
  â”‚ â””â•´[31md.txt[m â‡¡
  â•°â•´[34mlinked-src.html[m
